include/logging/log_core.h

```diff
@@ -422,7 +422,7 @@ static inline u32_t log_dynamic_source_id(struct log_source_dynamic_data *data)
 static inline __printf_like(1, 2)
 void log_printf_arg_checker(const char *fmt, ...)
 {
-
+	(void)fmt;
```

Comment:
Is this an argument unused warning? We have a dedicated macro for that. Can't remember it's name though ...

grep for UNUSED maybe.

Reply:
Ok, thanks. It seems to be ARG_UNUSED(). I updated the patch to utilize that instead.